packages/app/src/app/components/UploadProgress/Cube/elements.ts

```diff
@@ -9,35 +9,37 @@ const rotateAnimation = keyframes`
   }
 `;
 
-export const Cube = styled.div`
-  position: absolute;
-  width: ${props => props.size + 1}px;
-  height: ${props => props.size + 1}px;
-
-  * {
+export const Container = styled.div`
+  ${({ size }: { size: number }) => css`
     position: absolute;
-    width: ${props => props.size + 1}px;
-    height: ${props => props.size + 1}px;
-  }
+    width: ${size + 1}px;
+    height: ${size + 1}px;
+
+    * {
+      position: absolute;
+      width: ${size + 1}px;
+      height: ${size + 1}px;
+    }
+  `}
 `;
 
 export const Sides = styled.div`
-  ${({ noAnimation }) =>
-    !noAnimation &&
-    css`
-      animation: ${rotateAnimation} 1s linear infinite;
-    `};
-  transform-style: preserve-3d;
-  transform: rotateX(-37.5deg) rotateY(45deg);
+  ${({ size }: { size: number }) => css`
+    animation: ${rotateAnimation} 1s linear infinite;
+    transform-style: preserve-3d;
+    transform: rotateX(-37.5deg) rotateY(45deg);
 
-  * {
-    box-sizing: border-box;
-    background-color: rgba(242, 119, 119, 0.5);
-    border: ${({ size }) => size / 10}px solid white;
-  }
+    * {
+      box-sizing: border-box;
+      background-color: rgba(242, 119, 119, 0.5);
+      border: ${size / 10}px solid white;
+    }
+  `}
 `;
 
 export const Side = styled.div`
-  transform-origin: 50% 50%;
-  transform: ${({ rotate }) => rotate} translateZ(${({ size }) => size / 2}px);
+  ${({ rotate, size }: { rotate: string; size: number }) => css`
```

Comment:
@Saeris I think it's better to extract the `SideProps` here, just to be consistent